web/src/EducationBenchmarking.Web/Views/SchoolPlanningYear/Index.cshtml

```diff
@@ -1,38 +1,70 @@
+@using EducationBenchmarking.Web.Extensions
 @model EducationBenchmarking.Web.ViewModels.SchoolPlanViewModel
 @{
+
     ViewBag.Title = PageTitleConstants.SchoolPlanningYear;
-    var previousYear = Model.CurrentYearEnd - 1;
+    int previousYear;
+    int yearEnd;
+    string totalExpenditure;
+    string totalIncome;
+    string totalTeacherCosts;
+    string educationSupportStaffCosts;
+    decimal? totalNumberOfTeachersFte;
+
+    if (Model.PlanExists)
```

Comment:
This is incorrect - the figures on this screen should not be the previously entered values. Later in the journey they will be, but this step is about reusing the existing financial data.

Reply:
fixed - removed this in latest commit